src/ng/directive/form.js

```diff
@@ -276,10 +276,17 @@ function FormController(element, attrs, $scope, $animate, $interpolate) {
    * @description
    * Sets the form to its submitted state.
    */
-  form.$setSubmitted = function() {
+  form.$setSubmitted = function(reverse) {
```

Comment:
`reverse` is not really a good name. How about `setOnChild` or something similar?